src/irr/video/COpenGLRenderpassIndependentPipeline.h

```diff
@@ -0,0 +1,45 @@
+#ifndef __IRR_C_OPENGL_RENDERPASS_INDEPENDENT_PIPELINE_H_INCLUDED__
+#define __IRR_C_OPENGL_RENDERPASS_INDEPENDENT_PIPELINE_H_INCLUDED__
+
+#include "irr/video/IGPURenderpassIndependentPipeline.h"
+#include "COpenGLExtensionHandler.h"
+#include "COpenGLSpecializedShader.h"
+
+namespace irr {
+namespace video
+{
+
+class COpenGLRenderpassIndependentPipeline : public IGPURenderpassIndependentPipeline
```

Comment:
this class shouldn't be making the OpenGL pipeline object... since its per-thread, just store/compute the hash and refcount the stages etc. hooked up to it

Reply:
well it can help create, but nothing more

Reply:
@Crisspl ?

Reply:
ok

Reply:
@Crisspl hows the move going?

Reply:
will do that while making pipeline cache in CopenglDriver, in plans for tomorrow

Reply:
yea long ago